deploy/runtime/ui/PV System Design.json

```diff
@@ -8087,6 +8087,19 @@
             "sscVariableName": "",
             "sscVariableValue": ""
         },
+        "num_inverter_subhourly_clipping": {
```

Comment:
See my comment in the SSC PR review, but I think this value should be calculated in SSC rather than passed as a variable.